lang/en.yml

```diff
@@ -36,6 +36,7 @@ en:
   SilverStripe\Forms\ConfirmedPasswordField:
     ATLEAST: 'Passwords must be at least {min} characters long.'
     BETWEEN: 'Passwords must be {min} to {max} characters long.'
+    RANDOM_IF_EMPTY: 'If this is left blank then a random password will be automatically generated. The user will need to click the forgotten password link to generate a reset password email so they can set their own password.'
```

Comment:
> The user will need to click the forgotten password link to generate a reset password email so they can set their own password

This is all related to `Member` records - this shouldn't be on the `ConfirmedPasswordField` which could be used for who knows what, including API tokens, front-end users where perhaps "forgotten password link" functionality isn't implemented, etc.

Reply:
Changed to simply "If this is left blank then a random password will be automatically generated."